devel/ansible/roles/backend/templates/fmn.cfg.j2

```diff
@@ -6,3 +6,4 @@ OIDC_TOKEN_INFO_ENDPOINT="/TokenInfo"
 OIDC_CLIENT_ID="{{ oidc_client_id }}"
 OIDC_CLIENT_SECRET="{{ oidc_client_secret }}"
 SERVICES__FASJSON_URL=http://fasjson.tinystage.test/fasjson
+SERVICES__DISTGIT_URL=http://src.tinystage.test/
```

Comment:
You should probably leave out the last `/`, otherwise you'll have two slashes when you build the full URL in the client.